src/optionals/gravity_opt_env.c

```diff
@@ -124,14 +152,26 @@ void gravity_env_register(gravity_vm *vm) {
     gravity_vm_setvalue(vm, GRAVITY_CLASS_ENV_NAME, VALUE_FROM_OBJECT(gravity_class_env));
 }
 
+void gravity_env_register_args(gravity_vm *vm, uint32_t _argc, const char **_argv) {
+    gravity_value_t tmp[_argc];
+    for (int i = 0; i < _argc; ++i) {
+        tmp[i] = VALUE_FROM_CSTRING(NULL, _argv[i]);
```

Comment:
This feel a little hack-ish, is there a better way to append an item to a gravity_list_t?

Reply:
It should be rewritten as:
```swift
void gravity_env_register_args(gravity_vm *vm, uint32_t _argc, const char **_argv) {
    argc = _argc;
    argv = gravity_list_new(vm, argc);
    for (int i = 0; i < _argc; ++i) {
        gravity_value_t = VALUE_FROM_CSTRING(vm, _argv[i]);
        marray_push(gravity_value_t, argv, value);
    }
}
```